src/password/Create.vue

```diff
@@ -136,6 +138,17 @@ const required = (v, vm) => !!v || vm.$vuetify.lang.t('$vuetify.password.create.
 const minLength = (v, vm) => v.length >= vm.$root.idpConfig.passwordRules.minLength || vm.$vuetify.lang.t('$vuetify.password.create.tooShort', vm.$root.idpConfig.passwordRules.minLength)
 const maxLength = (v, vm) => v.length < vm.$root.idpConfig.passwordRules.maxLength || vm.$vuetify.lang.t('$vuetify.password.create.tooLong', vm.$root.idpConfig.passwordRules.maxLength)
 const strong = (v, vm) => vm.strength.score >= vm.$root.idpConfig.passwordRules.minScore || vm.$vuetify.lang.t('$vuetify.password.create.tooWeak')
+const options = {
+  useLevenshteinDistance: true,
```

Comment:
depending how this affects performance we may want to use a debouncer on this. See https://zxcvbn-ts.github.io/zxcvbn/guide/options/#uselevenshteindistance (you probably already did)

Reply:
Thanks, I missed that comment. Now to see if I can figure out how to add that to this implementation.

Reply:
I've been having a look at trying to implement it, and it seems to be fairly ugly to implement it (and it doesn't work well due to causing it to not send as frequently). Do we have a preference as to what route we go forward with this?

1. As is
2. Remove `useLevenshteinDistance`

Reply:
If the performance is ok I would say as is. I am not sure what the quantified benefit of using this feature is though.

Reply:
Looked around to see what it was exactly:

> The Levenshtein distance between two words is the minimum number of single-character edits required to change one word into the other.

So it looks like it makes it even more stringent as it would also check to see how close it is to a real word, making the passwords more secure.

For now I think I will remove it (I didn't notice any impact to performance, but I only tested it locally.) But it's something we should consider enabling later.

Reply:
I may be missing some of the history or context, but using real words in passwords is not inherently a problem. Passwords like "visaged-lawful-saith-air" are actually quite good.

Reply:
Yeah, I went ahead and removed it for now. We can have further discussions later if we want to add it back in.